applications/advanced-concepts/multi-engine-machine-translation.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Multi-engine machine translation
+---
+
+# Multi-engine machine translation
+
+**Multi-engine machine translation** is the process of evaluating and picking a machine translation engine. In multi-engine machine translation, different [providers](business/providers.md) translate the same input sample. The output versions are then evaluated automatically or by a human to select a provider.
+
+Multi-engine machine translation is challenging because it is subjective.
```

Comment:
I'd say the challenges are:
- consistency across segments
- complexity, latency and cost

i.e. accuracy challenges, vs practical implementation challenges